xml/html_body.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<fingerprints matches="html_body" protocol="http" database_type="service" preference="0.90">
+  <!-- HTML found in HTTP response bodies are matched against these patterns. -->
+
+  <fingerprint pattern="EDGE.DeviceModel = 'ER-X-SFP'">
+    <description>Ubiquiti EdgeRouter X-SFP</description>
+    <example>EDGE.DeviceModel = 'ER-X-SFP'</example>
```

Comment:
I recommend all examples be files here containing all of the content prior up to and just past the match. The reason for this is to avoid false positives as this fingerprint database grows over time.

Reply:
0.02 is that including the full html in the example is tough unless you use the on-disk case (the amount of escaping needed is horrendous otherwise).

Reply:
Sorry, I should have been clearer. This should use _*file based*_ examples, not encode them in the XML.

Reply:
@jvoisin See the bottom of the [Fingerprints](https://github.com/rapid7/recog#fingerprints) section for information on external examples.